datajoint/blob.py

```diff
@@ -251,6 +263,34 @@ def pack_recarray(self, array):
     def read_sparse_array(self):
         raise DataJointError('datajoint-python does not yet support sparse arrays. Issue (#590)')
 
+    def read_int(self):
+        return int(self.read_value('int64'))
+
+    @staticmethod
+    def pack_int(v):
+        return b"\x0a" + np.array(v, dtype='int64').tobytes()
```

Comment:
Is there a reason why we did not utilize decimal packing here? Python `int` are essentially boundless (memory-dependent). I believe `decimal` packing would be a closer representation as the length would be encoded.

Reply:
modified to support unbounded `int`